src/coll/error.rs

```diff
@@ -320,17 +332,19 @@ impl BulkWriteException {
         // Parse out any write errors.
         let w_errs = if let Some(&Bson::Array(ref errors)) = result.get("writeErrors") {
             if errors.is_empty() {
-                return Err(Error::ResponseError(
-                    "Server indicates a write error, but none were found.".to_owned()));
+                return Err(Error::ResponseError("Server indicates a write error, but none were \
+                                                 found."
+                    .to_owned()));
```

Comment:
Just pushed the changes to this branch. A lot of the instances of `.to_owned()` can't be changed due to them being called on `&String` rather than `&str`. Do you think it's worth trying to change those to `.clone()`?

I also decided to hold off on making the changes in the `v3_x` test directories pending the re-merging of them that we discussed offline, since it will take half as much work to do that after they're merged again.